tests/components/light/test_device_action.py

```diff
@@ -138,3 +180,19 @@ def calls(hass):
     hass.bus.async_fire("test_event3")
     await hass.async_block_till_done()
     assert hass.states.get(ent1.entity_id).state == STATE_ON
+
+    turn_on_calls = async_mock_service(hass, DOMAIN, "turn_on")
+
+    hass.bus.async_fire("test_brightness_increase")
+    await hass.async_block_till_done()
+
+    assert len(turn_on_calls) == 1
+    assert turn_on_calls[0].data["entity_id"] == ent1.entity_id
+    assert turn_on_calls[0].data["brightness_step_pct"] == 10
+
+    hass.bus.async_fire("test_brightness_decrease")
+    await hass.async_block_till_done()
+
+    assert len(turn_on_calls) == 2
+    assert turn_on_calls[1].data["entity_id"] == ent1.entity_id
+    assert turn_on_calls[1].data["brightness_step_pct"] == -10
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4972290)